### PR TITLE
Slider unit tests updated to use test-extras

### LINKED ISSUE #144: Slider: Update Unit Tests
Update Slider unit tests to use `test-extras`.

### PR DESCRIPTION
**Pull request by smhigley**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

Resolves #144 
Also updates string casting to consistently use template literals

### CODE REVIEW

File: src/slider/Slider.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export default class Slider extends SliderBase<SliderProperties> {
 		const percentValue = (value - min) / (max - min) * 100;
 
 		// custom output node
-		const outputNode = output ? output(value) : value + '';
+		const outputNode = output ? output(value) : `${value}`;
```
Comment (kitsonk) [inline src/slider/Slider.ts]: both of those are ugly... `String(value)` would be so much better... 🙉

Comment (agubler) [reply src/slider/Slider.ts]: Template literals for the win 🙈

Comment (mwistrand) [reply src/slider/Slider.ts]: Personally, I prefer `value + []` for all string casting 🙊

Comment (smhigley) [reply src/slider/Slider.ts]:
```
`${value}` + String('')
```

Comment (kitsonk) [reply src/slider/Slider.ts]:
```js
typeof value === 'string' ? value : value.toString()
```

Comment (kitsonk) [reply src/slider/Slider.ts]:
```ts
[ '' ] + [ value ]
```

🤡

Comment (smhigley) [reply src/slider/Slider.ts]:
```
Math.random() > 0.5 ? `${value + ''}` : String(value);
```

Comment (bitpshr) [reply src/slider/Slider.ts]: Template literals are fastest now (we had a JSPerf page going around another PR at some point.)

Comment (kitsonk) [reply src/slider/Slider.ts]: Ok, fine, though we shouldn't always overvalue performance for code clarity though.

Comment (kitsonk) [reply src/slider/Slider.ts]: Also, there is no speed advantage as long as we continue to emit to ES5, as the down emit would be:

```js
"" + value
```

But even that is more performant than `String(value)`

Comment (agubler) [reply src/slider/Slider.ts]: Main reason I like like template literals is that is for consistency as we use them in various scenarios across all our packages. One of which is to effectively stringify a variable :)

Comment (kitsonk) [reply src/slider/Slider.ts]: Using literals to stringify variables, as long as we are consistent is sufficient and clear.  It is a darn sight better than `value + ''`.  Refs: dojo/meta#198

**Pull request merged**